frontend/src/components/quize-list/input-range/input-range.scss

```diff
@@ -0,0 +1,103 @@
+.range-slider,
+.range-ticks,
+.label-range{
+  display: flex;
+}
+
+.range-slider {
+  flex-flow: column nowrap;
+  width: 27em;
+  box-sizing: border-box;
+  font-size: 2em;
+}
+
+.range-input,
+.range-item {
+  border-radius: 0.75em;
+  overflow: hidden;
+  position: relative;
+  height: 1.5em;
+}
+
+.range-input {
+  background-color: transparent;
+  font-size: 1em;
+  box-shadow: 0.3em 0.3em 0.4em #c1c2c5 inset, -0.3em -0.3em 0.4em #fff inset;
+  display: block;
+  padding: 0 0.1em;
+  width: 100%;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+}
+
+.range:focus {
+  outline: transparent;
+}
+
+.range-ticks {
+  font-size: 1em;
+  justify-content: space-between;
+  align-items: center;
+  pointer-events: none;
+  position: absolute;
+  top: 0.04em;
+  left: 0.75em;
+  width: calc(100% - 1.9em);
+  height: 100%;
+}
+
+.range-tick,
+.range-tick-text {
+  display: inline-block;
+}
+
+.range-tick {
+  color: #454954;
+  font-size: 0.5em;
+  text-align: center;
+  width: 0;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  user-select: none;
+}
+
+.range-tick-text {
+  transform: translateX(-50%);
+}
+
+.data-range {
+  display: flex;
+  height: 3em;
+  width: 3em;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  font-size: 0.5em;
+  color: #fff;
+  position: absolute;
+  pointer-events: none;
+  top: 0;
+  z-index: 2;
+  background-color: #463da8;
+  border: 0;
+  border-radius: 50%;
+  box-shadow: -0.3em -0.3em 0.5em #463da8 inset, 0 -0.2em 0.2em 0 rgba(0,0,0,0.2), 0.3em 0.5em 0.8em  rgba(0,0,0,0.3);
+}
+
+
+@media screen and (max-width: 768px) {
```

Comment:
Use bootstrap for breakpoints - https://getbootstrap.com/docs/5.0/layout/breakpoints/#min-width
Also try to use our default font-size

And if you can use bootstrap classes instead of custom styles